frontend/libs/erxes-ui/src/hooks/use-cursor-pagination.tsx

```
@@ -0,0 +1,40 @@
+  useEffect(() => {
+    if (scrollRef.current) {
+      if (distanceFromBottomRef.current && isFetchBackward) {
+        scrollRef.current.scrollTop =
+          scrollRef.current.scrollHeight - distanceFromBottomRef.current;
+        distanceFromBottomRef.current = 0;
+        setIsFetchBackward(false);
+      } else if (!isUndefinedOrNull(hasPreviousPage) && offset) {
+        scrollRef.current.scrollTop = hasPreviousPage && !loading ? offset : 0;
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)